packages/Textarea/README.md

```diff
@@ -22,56 +22,51 @@ npm install @paprika/textarea
+| isReadOnly   | bool                                                                                | false    | false                      | If true it makes the textarea read only.                                                                                                       |
+| maxHeight    | [number,string]                                                                     | false    | 300                        | The maximum height of the textarea.                                                                                                            |
+| minHeight    | [number,string]                                                                     | false    | 80                         | The minimum / default height of the textarea.                                                                                                  |
+| onChange     | func                                                                                | false    | () => {}                   | Callback to be executed when the textarea value is changed. Receives the onChange event as an argument. Required when component is controlled. |
+| size         | [ Textarea.types.size.SMALL, Textarea.types.size.MEDIUM, Textarea.types.size.LARGE] | false    | Textarea.types.size.MEDIUM | The size of the textarea input (font size).                                                                                                    |
+| value        | string                                                                              | false    | undefined                  | The value inside of the textarea input. Defining this prop will make this a controlled component. Do not use in conjunction with defaultValue. |
 
 <!-- end: Autogenerated - do not modify -->
 <!-- content -->
 
 ### Usage
 
-The Textarea can be used as a controlled or an un-controlled component.
+The Textarea can be used as a controlled or uncontrolled component.
 
-To use it as controlled:
+To use it as a controlled comnponent:
 
 ```js
 import Textarea from "@paprika/textarea";
 ...
-const [val, setVal] = React.useState("Hello world");
-return (
-  <Textarea
-    value={val}
-    onChange={e => {
-      setVal(e.target.value);
-    }}
-  />;
-);
+const [value, setValue] = React.useState("Hello world");
+...
+<Textarea
+  value={value}
+  onChange={event => { setValue(event.target.value) }}
+/>
 ```
 
-To use it as an un-controlled component:
+To use it as an uncontrolled component:
 
 ```js
 import Textarea from "@paprika/textarea";
 ...
-return (
-  <Textarea
-    defaultValue="Hello world"
-    onChange={e => {
-      console.log(`The new value is ${e.target.value}`);
-    }}
-  />
-);
+<Textarea
+  defaultValue="Hello world"
```

Comment:
To comply with the example of uncontrrolled components in react documentation I think wee should do something like this:


```js
const refTextarea = React.useRef(null)

<Textarea ref={refTextarea} defaultValue="hi!" />

refTextarea.current // hi!

```

using the onChange will imply that you have to store dom value in some state to do something with it or to submit it.

Reply:
Good point.  I think that makes sense, but I'll want to update the Uncontrolled story as well and also update the `<Select>` and `<Input>` to make them all consistent.